src/nxdoc/nuxeo-server/platform-services/comments.md

```diff
@@ -20,7 +20,73 @@ Comments (and so annotations) are linked to a specific document (their parent).
 
 ## Core Implementation
 
-### Storage
+### Current implementation - TreeCommentManager
+
```

Comment:
I am wondering if we should specify `TreeCommentManager`, I mean perhaps give more context and explain that a comment is stored under the commented document which represents the Tree Storage structure.

Reply:
Is this what you mean ? Or do you think more should be said ?
https://github.com/nuxeo/doc.nuxeo.com-content/pull/1486/commits/99aa2e90c8e3562dd6b87e2d8be023d2d87e3a5c#diff-d0ac791dfc7f98240e792b1fc40a612eR52-R57

Reply:
My idea is here:
- Should we specify the `TreeCommentManager` and if it is the case, I think we should link to the class.
- If we specify it, it means we have others implementation which are not explained here. what if we say `Comment Are stored as a Tree Structure`. 

Which is some confusing for me is we talk at the beginning of the implementation and the details of it is done later (99aa2e9#diff-d0ac791dfc7f98240e792b1fc40a612eR52-R57)

Reply:
about the bullet points:
I wish I could link the classes but http://community.nuxeo.com/api/nuxeo/latest/javadoc/ doesn't have the `TreeCommentManager` actually. It only has the `PropertyCommentManager`. It's due to the fact the `TreeCommentManager` is only available in HF and Master. So I end up with just the name.

I specified Tree and Property to explicit the fact we are not talking about Property and Relation. I think it's more relevant than "current" and "legacy" ?

About the explanations concerning the implementation, they come in the same position as for the previous implementation (I adapted the existing doc) but if you have a better order in mind, I'm open to it !